nextflow/modules/contig_search_taxonomy/diamond_blastp_workflow.nf

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl=2
+
+include { DIAMOND } from './process/diamond.nf'
+
+workflow DIAMOND_BLASTP {
+    take:
+      orfs
+
+    main:
+      DIAMOND(orfs)
+    emit:
+      blastp_table = DIAMOND.out.blastp_table // output '${orfs.simpleName}.blastp.tsv'; BLAST fmt 6 table
+}
+
```

Comment:
I think it was due to future-proof it so it could be used more than once within the same workflow context. But that BLAST call needs to be generalized before that

Reply:
Fixed in https://github.com/KwanLab/Autometa/commit/604b49e82ebfb690929ca47a96c27d881d8e7976